doc: add link to VSCode config in testing

trivikr opened the pull request:
Refs: https://github.com/orgs/nodejs/teams/collaborators/discussions/58

This config will help contributors to run/debug tests on VSCode

<!--
Thank you for your pull request. Please provide a description above and review
the requirements below.

Bug fixes and new features should include tests and possibly benchmarks.

Contributors guide: https://github.com/nodejs/node/blob/master/CONTRIBUTING.md
-->

##### Checklist
<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->

- [x] documentation is changed or added
- [x] commit message follows [commit guidelines](https://github.com/nodejs/node/blob/master/doc/guides/contributing/pull-requests.md#commit-message-guidelines)

Trott commented on BUILDING.md:
```diff
@@ -259,6 +259,10 @@ $ ./node ./test/parallel/test-stream2-transform.js
 
+If you're on VSCode editor, you can use launch config defined in 
+[node-vscode-config](https://github.com/nodejs/node-vscode-config) repo to 
+run/debug tests.
```
Maybe wording closer to this?

```markdown
If you use Visual Studio Code, you can use
[node-vscode-config](https://github.com/nodejs/node-vscode-config) to
run/debug tests.
```

trivikr commented:
Updates about node-vscode-config:
* Issue for transfer of ownership was created at https://github.com/nodejs/admin/issues/260
* PR to add Code Of Conduct is created at https://github.com/nodejs/node-vscode-config/pull/2
* PR to add MIT License is created at https://github.com/nodejs/node-vscode-config/pull/3

refack commented:
> I'd much prefer moving this kind of stuff into a separate guide that can detail the use of multiple tools.

👍

trivikr commented:
@jasnell Steps taken:
* Updated the names suggested for the repo in https://github.com/nodejs/node-vscode-config/issues/1
* Updated this PR with the latest popular name (i.e. `node-code-ide-configs`)
* Created PR to move VSCode specific configs to .vscode folder https://github.com/nodejs/node-vscode-config/pull/4

trivikr commented:
Since I ran out of time, I forgot to show the VSCode launch configs being used in my [NodeConfEU](https://www.nodeconf.eu/) talk today.

Can we push this code so that new contributors don't get stuck finding out the VSCode launch config while contributing to Node?
On Google, searching for [node VSCode Config](https://www.google.ie/search?q=node+vscode+config) shows which VSCode Config to use while debugging Node.js, and not Node.js core.

trivikr commented:
Added tsc-agenda label, for discussions related to following questions:
* What should be the name for new repo which stores editor configs?
  * Issue created at https://github.com/nodejs/node-vscode-config/issues/1, suggestions as of 11/09 are `node-editor-configs` and `node-code-ide-configs`
* Where should we link the repo from Node.js core? Should it link from BUILDING.md or CONTRIBUTING.md or somewhere else?

Follow up required on PRs:
* Adding Code of Conduct https://github.com/nodejs/node-vscode-config/pull/2
* Adding MIT License https://github.com/nodejs/node-vscode-config/pull/3
* Move VSCode configs to .vscode folder https://github.com/nodejs/node-vscode-config/pull/4

trivikr commented:
Removed tsc-agenda label as we agreed on name `node-code-ide-configs` in https://github.com/nodejs/TSC/issues/633

trivikr commented:
@Trott @jasnell This PR is ready for review.

trivikr commented:
Landed in e55d65384d3feb3dd1bb7341a23766663fbcdba6
